modules/troubleshooting/pages/index.adoc

```diff
@@ -318,6 +318,34 @@ linking and services:: See the `skupper-service-controller-*-events.txt` file to
 
 --
 
+// Type:procedure
+[id='network-performance']
+== Improving network performance
+
+If you encounter network performance issues, you can scale Skupper to address those concerns.
+For example, use this procedure to increase network throughput, and if you have many clients, network latency.
+
+. Note all configuration and delete your existing site:
```

Comment:
Maybe say that at present changing this requires that you recreate the site. (In future we should make it so that you don't need to).

Reply:
I've put a note at start of procedure